### PR TITLE
Update to reflect dev preview enablement and targeted release enrollment

### PR DESCRIPTION
**Pull request by JoshuaPartlow**
Exact update related to developer preview enablement when tenant is enrolled for targeted releases is still being determined, but creating a Draft PR we can iterate on. Will change to regular PR when ready for review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/996f8672c132ad111adee204f4a1013b640b50dd/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ba91333-dd37-4917-a1c5-e5ed31d799c9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/996f8672c132ad111adee204f4a1013b640b50dd/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 89 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f5484cd9-1fbb-4722-ba2a-5fe3f1f095d6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/29dbbaf53f35f33661be21a4052ff1abc97bbc59/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35654e29-c5d3-47e9-86c5-e5d61a19dd9b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/29dbbaf53f35f33661be21a4052ff1abc97bbc59/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 89 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2026f27b-07f4-4ac2-b728-ee1e4f7aac10) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/dev-preview/developer-preview-intro.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Using an app that contains developer preview features may cause clients that hav
 
 ### On a desktop or web client
 
+> [!NOTE]
+> If your tenant is enrolled for *Microsoft 365 Targeted Releases* developer preview is automatically enabled and you will not see the option to switch to developer preview.
```
Comment (erikadoyle) [inline msteams-platform/resources/dev-preview/developer-preview-intro.md:29]: Let's add a link to:

https://learn.microsoft.com/microsoft-365/admin/manage/release-options-in-office-365

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c5244e43028f6b2c3b7fb5ccb994fa27ae3c47d2/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7104e8ab-2344-4bea-8e10-670dd8b23e18) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c5244e43028f6b2c3b7fb5ccb994fa27ae3c47d2/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 89 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6aa3e8ed-c0b1-4443-8705-a9adf59bec1c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39fbfec6b1d78090f0fe1ef0f8a926ac5eb8d85d/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1750e498-5741-41fa-b5c4-98232287eccc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39fbfec6b1d78090f0fe1ef0f8a926ac5eb8d85d/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 89 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0574c043-81ec-4a00-b214-3329e08a73cc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by JoshuaPartlow** [comment]
Small changes about targeted releases and developer preview enablement. @erikadoyle

**Comment by JoshuaPartlow** [comment]
@nwojod-MSFT for review.

**Review by nwojod-MSFT — DISMISSED** [review]
Changes looks good to me.

**Review by erikadoyle — DISMISSED** [review]
Looks good!

**Comment by erikadoyle** [comment]
@v-ypalikila would you mind helping review/merge? It fixes #7171 and helps unblock ISV partners. Thank you!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c9bf037d669c58dd5962453087efd6bddf75a4a1/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5704f108-cace-4568-b67a-10efb2b45c58) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c9bf037d669c58dd5962453087efd6bddf75a4a1/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 89 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94559c85-4ea5-41e8-98e4-bbd88a98d096) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by erikadoyle** [comment]
Thanks @v-ypalikila for prepping this for merge--is it ready?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/14269e3e508e77c63e9fc182e51f792059374858/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 89 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7efd7f3a-751c-43da-9306-52ab6ed5e504) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/14269e3e508e77c63e9fc182e51f792059374858/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 89 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd7b14ec-10b8-452f-b690-9a4972825f44) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9ab86b1e39c6f699e8e259fdce87b43d6240a64/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 92 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cdf82bd2-d9fa-4fa4-988e-d80424b36e4d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/dev-preview/developer-preview-intro.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f9ab86b1e39c6f699e8e259fdce87b43d6240a64/msteams-platform%2Fresources%2Fdev-preview%2Fdeveloper-preview-intro.md) | 91 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/94c3496d-61ce-4541-b22b-d756d8d5cc2f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**